src/types/User.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-unused-vars */
```

Comment:
can we get rid of this eslint-disable rule?

Reply:
Without this the `UserStatus` enum gets a `no-unused-vars` error. We actually do this in all our `/type` files for enums. We can probably figure out how to avoid this with some tslint configuration if we want to change it!